core/error/error_reply.py

```diff
@@ -19,3 +19,22 @@ async def error_reply(ctx: Context,
     user: Union[Member, User] = ctx.author
     await ctx.reply(embed=embed, content=content, delete_after=delete_after)
     logger.error(f'User="{user.name}#{user.discriminator}({user.id})", Command="{command}", Cause="{cause}"')
+
+
+async def startup_error_reply(
+        bot: Bot,
+        title: str,
+        cause: str,
+        solution: str):
+    embed = Embed(title=title, description="Please notify the admin immediately")
+    embed.add_field(name="Cause", value=cause, inline=False)
+    embed.add_field(name="Solution", value=solution, inline=False)
+
+    guild: Guild = bot.guilds[0]
+    text_channels: list[TextChannel] = guild.text_channels
+
+    for channel in text_channels:
+        permissions: Permissions = channel.permissions_for(guild.get_member(bot.user.id))
+        if permissions.read_messages and permissions.send_messages:
+            await channel.send(embed=embed)
```

Comment:
Why send it in every possible channel?

Reply:
As you can see in line 40 (https://github.com/Anthony271828/HM-DiscordBot/pull/32/files#diff-f546f1102df14e5a98a484b02643aeb68053bb78a15a73b1d6b01a4f425c80c8R40)
Means that the first usable chat is used, not all!

Reply:
oh yes sorry didn't see